pom.xml

```diff
@@ -94,6 +94,11 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
?

Reply:
oh whoops, this was from when Elliot and I were debugging the join tables issue, will remove